lib/utils.js

```diff
@@ -5,7 +5,7 @@ const isOvsxPublishEnabled = () => {
 };
 
 const isVscePublishEnabled = () => {
-  return 'VSCE_PAT' in process.env;
+  return 'VSCE_PAT' in process.env || 'VSCE_AZURE_CREDENTIALS' in process.env;
```

Comment:
I've got to say I'm tempted to say VSCE_USE_AZURE_CREDENTIALS would be a better name, but it's not a strong opinion.

Do you have one?